input.go

```diff
@@ -18,6 +18,19 @@ import (
 	"github.com/hajimehoshi/ebiten/internal/ui"
 )
 
+// CopyChars copies "printable" runes read from the keyboard each update to buf.
+//
+// CopyChars represents the environment's locale-dependent translation of keyboard
+// input to Unicode characters.
+//
+// IsKeyPressed is based on a mapping of device codes to input device keys.
+// "Control" and modifier keys should be handled with IsKeyPressed.
+//
+// This function is concurrent-safe.
+func CopyChars(buf []rune) int {
```

Comment:
Sorry but I wanted you to also revert the signature to `InputChars() []rune`,  since I feel like taking a slice and filling it is still not usual. Thank you.